harness/tests/cli/test_user.py

```diff
@@ -9,9 +10,89 @@
 from tests.cli import util
 
 
+def test_user_create_with_password_flag() -> None:
```

Comment:
It's unclear to me what this test is supposed to test. At the moment, I think it tests:
1. the tests that are inserted implicitly by `standard_cli_rsps` (like that `version` is called or something)
2. That the `users` endpoint is called with `isHashed == True`, `user = {<user struct>}` and `password = <salted_hashed_password>`.

when the user runs
```
det user create test-user-1 <long password>
```

I don't think I'd write this test if this was my PR because it seems to be more about the internal implementation than the function's behavior. But if you want to argue that the internal API call is a proxy for satisfying the function's behavior then I won't argue with you very hard.

If that's the case, my request is that you rename the test to be explicit about the behavior you're testing.

Reply:
This is mostly for branch coverage -- the command behaves slightly differently (becomes noninteractive) if the flag is provided. But to be fair, this test doesn't set up an input pipe to make sure it's ignored. I'm not going to fight hard to keep the test around given basically everything you said was accurate.

Reply:
Thanks for splitting the tests up. I like how they read.